oak/client/manager_client.h

```diff
@@ -30,20 +31,27 @@ class ManagerClient {
 
   // TODO: Return StatusOr<::oak::CreateApplicationResponse>.
   CreateApplicationResponse CreateApplication(const std::string& module_bytes) {
+    return CreateApplication(module_bytes, true, "");
+  }
+
+  CreateApplicationResponse CreateApplication(const std::string& module_bytes, bool enable_logging,
```

Comment:
FWIW I don't think the enable_logging boolean here will be sufficient long term. The way that logging may or may not be allowed in an application depends on the policies associated with the data that it is processing, rather than on a per-application configuration. Anyway probably fine to have around for convenience for the time being.

Reply:
Yeah, I'd expect this to change later but for now I wanted the examples to sync with the `ApplicationConfiguration` proto.